pkg/cvo/metrics.go

```diff
@@ -114,6 +120,10 @@ penultimate completed version for 'completed'.
 			Name: "cluster_version_operator_update_retrieval_timestamp_seconds",
 			Help: "Reports when updates were last successfully retrieved.",
 		}, []string{"name"}),
+		clusterVersionConditionalUpdatesRecommendedConditions: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "cluster_version_conditional_updates_recommended_conditions_seconds",
```

Comment:
nit: I realize this diverges from precedent like `cluster_operator_conditions`, but personally, I think this reads better with plural -> singular, like `cluster_version_conditional_update_recommended_condition_seconds`, because each time-series is talking about a single condition for a single update.

Reply:
Called `cluster_version_conditional_update_condition_seconds` now